File: src/Mobile_Customer/lib/features/home/domain/entities/store_banner.dart
class StoreBanner {
  final int id;

  final String imageUrl;
  final String? title;
  final String? description;

  final int sortOrder;
  final bool isActive;

  const StoreBanner({
    required this.id,
    required this.imageUrl,
    this.title,
    this.description,
    required this.sortOrder,
    required this.isActive,
  });
}

File: src/Backend_API/BeautyBookingSystem.Domain/Entities/StoreBanner.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BeautyBookingSystem.Domain.Entities
{
    public class StoreBanner
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public int StoreId { get; set; }

        [Required]
        [MaxLength(500)]
        public string ImageUrl { get; set; } = string.Empty;

        [MaxLength(255)]
        public string? Title { get; set; }

        [MaxLength(500)]
        public string? Description { get; set; }

        public int SortOrder { get; set; } = 0;

        public bool IsActive { get; set; } = true;

        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;

        public DateTime? UpdatedAt { get; set; }

        [ForeignKey("StoreId")]
        public Store Store { get; set; } = null!;
    }
}